manufacturing-ai-suite/industrial-edge-insights-time-series/apps/weld-anomaly-detection/ingestor-data/weld-anomaly-detection.csv

```diff
@@ -1,338 +1,2263 @@
--0.02,13.66,20.74,165.95,560,20.91
--0.02,13.62,20.74,169.84,560,20.94
--0.02,13.62,20.74,166.81,560,21.19
--0.02,13.59,20,167.6,560,21.14
-0.05,13.59,20.01,164.34,560.16,21.29
-0.05,13.54,20.01,161.59,560.16,20.93
-0.05,13.53,20.01,176.03,560.33,20.98
-0.05,13.56,20.02,158.5,560.49,20.98
-0.05,13.56,20.02,165.4,560.66,20.76
-0.05,13.54,20.02,169.41,560.66,20.81
-0.05,13.56,20.02,168,560.66,20.79
-0.05,13.57,20.03,167.06,560.82,21.09
-0.05,13.53,19.34,167.11,560.82,20.89
-0.03,13.55,19.34,168.21,560.98,21.16
-0.03,13.54,19.34,169.35,560.98,21.12
-0.03,13.55,19.34,165.18,560.98,21.29
-0.03,13.53,19.34,160.54,560.98,21.23
-0.03,13.53,19.34,165.16,560.98,21.28
-0.03,13.52,19.34,166.13,560.98,20.62
-0.03,13.53,19.34,166.75,560.98,20.17
-0.03,13.53,19.34,170.51,560.98,20.91
-0.03,13.53,18.7,168.59,560.98,20.76
-0.03,13.53,18.7,168.55,560.98,21.14
-0.03,13.5,18.7,170.28,560.98,21
-0.03,13.48,18.7,167.75,560.98,21.21
-0.03,13.48,18.7,168.93,560.98,21.16
-0.03,13.48,18.7,161.45,560.98,21.09
-0.03,13.48,18.7,165.44,560.98,20.48
-0.03,13.48,18.7,162.54,560.98,20.67
-0.03,13.48,18.7,165.43,560.98,20.67
-0.03,13.48,18.1,164.49,560.98,20.61
-0.01,13.45,18.1,169.06,560.98,21.01
-0.01,13.46,18.1,166.73,560.98,20.77
-0.01,13.46,18.15,168.96,562.62,20.97
-0.01,13.47,18.36,168.7,569.02,20.9
-0.01,13.44,18.61,177.14,576.89,21.18
-0.01,13.43,18.83,169.48,583.61,20.96
-0.01,13.42,18.83,168.2,583.61,21.13
-0.01,13.38,18.83,166.66,583.61,21.38
-0.01,13.4,18.24,168.24,583.61,21.23
--0.04,13.39,18.24,165.6,583.61,20.85
--0.04,13.4,18.38,169.19,588.2,21.16
--0.04,13.37,18.52,70.33,592.62,33.72
--0.04,13.37,18.54,0,593.28,0
--0.04,13.38,18.55,0,593.44,0
--0.04,13.4,18.55,0,593.61,0
--0.04,13.39,18.55,0,593.61,0
--0.04,13.4,18.55,0,593.61,0
--0.04,13.43,18.56,0,593.77,0
-0.08,13.4,17.99,0,593.77,0
-0.08,13.4,18,0,593.93,0
-0.08,13.37,18,0,593.93,0
-0.08,13.38,18,0,593.93,0
-0.08,13.37,18,0,593.93,0
-0.08,13.39,18,0,593.93,0
-0.08,13.35,18,0,593.93,0
-0.08,13.34,18,0,593.93,0
-0.08,13.32,18,0,593.93,0
-0.69,13.28,17.47,0,593.93,0
-0.69,13.17,17.47,0,593.93,0
-0.69,13.04,17.47,0,593.93,0
-0.69,12.83,17.47,0,593.93,0
-0.69,12.84,17.47,0,593.93,0
-0.69,12.18,17.47,0,593.93,0
-0.69,12.14,17.47,0,593.93,0
-0.69,11.79,17.47,0,593.93,0
-0.69,11.48,16.97,0,593.93,0
-1.38,11.06,16.97,0,593.93,0
-1.38,10.63,16.97,0,593.93,0
-1.38,10.13,16.97,0,593.93,0
-1.38,9.63,16.97,0,593.93,0
-1.38,9.15,16.97,0,593.93,0
-1.38,8.64,16.97,0,593.93,0
-1.38,8.21,16.97,0,593.93,0
-1.38,7.64,16.97,0,593.93,0
-1.38,7.2,16.5,0,593.93,0
-1.97,6.74,16.5,0,593.93,0
-1.97,6.34,16.5,0,593.93,0
-1.97,5.9,16.5,0,593.93,0
-1.97,5.54,16.5,0,593.93,0
-1.97,5.16,16.5,0,593.93,0
-1.97,4.75,16.5,0,593.93,0
-1.97,4.45,16.5,0,593.93,0
-1.97,4.08,16.5,0,593.93,0
-1.97,3.8,16.05,0,593.93,0
-2.46,3.52,16.05,0,593.93,0
-2.46,3.27,16.05,0,593.93,0
-2.46,3,16.05,0,593.93,0
\ No newline at end of file
+1.56,-0.11,0,0,3.44,0
+1.56,-0.09,0,0,3.61,0
+1.56,-0.13,0,0,3.77,0
+1.56,-0.12,0,0,4.1,0
+1.56,-0.13,0,0,4.26,0

```

Comment:
Hey @sathyendranv ,
I am thinking it would be better to have the diff CSVs or have some sort of demarcation in this csv to evaluate the model performance, will discuss later.